package.json

```diff
@@ -2,15 +2,13 @@
   "name": "jscs-config-seegno",
```

Comment:
I'm +1 on that, even though jscs supports all https://github.com/jscs-dev/node-jscs/blob/2481d632053dc0d5ca8020741e0bae4b14d818ed/lib/config/node-configuration.js#L113